CONTRIBUTING.md

```diff
@@ -43,15 +49,15 @@ Agreement. You can do so by going to https://cla.salesforce.com/sign-cla.
 
 ### Merging `develop` into `main`
 
-- When a development cycle finishes, the content of the `develop` branch will become the `master` branch
+- When a development cycle finishes, the content of the `develop` branch will become the `main` branch
```

Comment:
...finishes, the content of the `develop` branch becomes the `main` branch. (end with period?)